### PR TITLE
nov dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-november-2021.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+The Indexing Capability Plugin has received numerous fixes, major and minor. One exciting aspect is that it is now interoperable with Metalnx and the NIEHS developed global-search endpoint. Expect a release of Indexing Capability 4.2.10.1 very soon.
+
+The C++ REST API and Python iRODS Client have grown full support for tickets. We are actively preparing releases for both projects.
+
+We've finally seen NFSRODS successfully transfer a file in parallel. This is all possible due to Jargon's newly added support for replica access tokens and multi-1247 parallel I/O. We have a few more things to work out for NFSRODS, so stay on the lookout for the next release. It's going to be so good!
+
+Last but not least, there's been some discussion around replacing imcoll and the structured file APIs. Take a look at the Discussion section for information around that.
+
+Well, that wraps up this month's update. Have some turkey and we'll see you next month!
```
Comment (korydraughn) [inline content/posts/irods-development-update-november-2021.md:22]: not sure about the "Have some turkey" part.

@trel what do you think?

Comment (trel) [reply content/posts/irods-development-update-november-2021.md:22]: agreed - just a US holiday.   maybe something about how we're running fast to get things out by the end of the year.

rest looks good.

Comment (korydraughn) [reply content/posts/irods-development-update-november-2021.md:22]: awesome. i will adjust that last statement.

Comment (korydraughn) [reply content/posts/irods-development-update-november-2021.md]: updated.

**Pull request merged**